GitUI/Editor/Diff/HighlightInfo.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace GitUI.Editor.Diff;
+
+public struct HighlightInfo
+{
+    public int docOffset;
+    public int length;
+    public Color? backColor;
+    public Color? foreColor;
```

Comment:
No public fields please.

Reply:
This is internal to AnsiEscapeUtilities (maybe in Grep too in the follow up), it should be an internal class. To use a public class with properties here is just annoying...